[EVS] `opentelekomcloud_blockstorage_volume_v2` throws panic

When running `TestAccCBRVaultV3_basic`, I've faced the following:
```
=== RUN   TestAccCBRVaultV3_basic
panic: tags: must be a map

goroutine 317 [running]:
github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema.(*ResourceData).Set(0xc000548280, 0x1d41b09, 0x4, 0x1a6f100, 0xc000a00a50, 0x0, 0x0)
	/home/akachurin/go/pkg/mod/github.com/hashicorp/terraform-plugin-sdk/v2@v2.7.0/helper/schema/resource_data.go:230 +0x498
github.com/opentelekomcloud/terraform-provider-opentelekomcloud/opentelekomcloud/services/evs.resourceBlockStorageVolumeV2Read(0x1f59e98, 0xc00100a480, 0xc000548280, 0x1d36180, 0xc0000c4000, 0x0, 0x0, 0x0)
	/home/akachurin/GolandProjects/terraform-provider-opentelekomcloud/opentelekomcloud/services/evs/resource_opentelekomcloud_blockstorage_volume_v2.go:294 +0x1c31
github.com/opentelekomcloud/terraform-provider-opentelekomcloud/opentelekomcloud/services/evs.resourceBlockStorageVolumeV2Create(0x1f59e98, 0xc00100a480, 0xc000548280, 0x1d36180, 0xc0000c4000, 0x0, 0x0, 0x0)
	/home/akachurin/GolandProjects/terraform-provider-opentelekomcloud/opentelekomcloud/services/evs/resource_opentelekomcloud_blockstorage_volume_v2.go:232 +0x18b7
github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema.(*Resource).create(0xc0003022a0, 0x1f59e98, 0xc00100a480, 0xc000548280, 0x1d36180, 0xc0000c4000, 0x0, 0x0, 0x0)
	/home/akachurin/go/pkg/mod/github.com/hashicorp/terraform-plugin-sdk/v2@v2.7.0/helper/schema/resource.go:330 +0x35d
github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema.(*Resource).Apply(0xc0003022a0, 0x1f59e28, 0xc000a30340, 0xc00019a2a0, 0xc0012da340, 0x1d36180, 0xc0000c4000, 0x0, 0x0, 0x0, ...)
	/home/akachurin/go/pkg/mod/github.com/hashicorp/terraform-plugin-sdk/v2@v2.7.0/helper/schema/resource.go:456 +0xb7a
github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema.(*GRPCProviderServer).ApplyResourceChange(0xc000a44798, 0x1f59e28, 0xc000a30340, 0xc000f5a140, 0x0, 0x0, 0x0)
	/home/akachurin/go/pkg/mod/github.com/hashicorp/terraform-plugin-sdk/v2@v2.7.0/helper/schema/grpc_provider.go:955 +0x14eb
github.com/hashicorp/terraform-plugin-go/tfprotov5/server.(*server).ApplyResourceChange(0xc000d18600, 0x1f59e28, 0xc000a30340, 0xc00019a000, 0x0, 0x0, 0x0)
	/home/akachurin/go/pkg/mod/github.com/hashicorp/terraform-plugin-go@v0.3.0/tfprotov5/server/server.go:332 +0x182
github.com/hashicorp/terraform-plugin-go/tfprotov5/internal/tfplugin5._Provider_ApplyResourceChange_Handler(0x1ccdbc0, 0xc000d18600, 0x1f59ed0, 0xc000eb8300, 0xc00100a0c0, 0x0, 0x0, 0x0, 0x0, 0x0)
	/home/akachurin/go/pkg/mod/github.com/hashicorp/terraform-plugin-go@v0.3.0/tfprotov5/internal/tfplugin5/tfplugin5_grpc.pb.go:380 +0x3d9
google.golang.org/grpc.(*Server).processUnaryRPC(0xc0000c4c40, 0x1f63e98, 0xc000445b00, 0xc0006f4100, 0xc0008a0450, 0x274f700, 0x0, 0x0, 0x0)
	/home/akachurin/go/pkg/mod/google.golang.org/grpc@v1.32.0/server.go:1194 +0x105d
google.golang.org/grpc.(*Server).handleStream(0xc0000c4c40, 0x1f63e98, 0xc000445b00, 0xc0006f4100, 0x0)
	/home/akachurin/go/pkg/mod/google.golang.org/grpc@v1.32.0/server.go:1517 +0x9f9
google.golang.org/grpc.(*Server).serveStreams.func1.2(0xc0000f8020, 0xc0000c4c40, 0x1f63e98, 0xc000445b00, 0xc0006f4100)
	/home/akachurin/go/pkg/mod/google.golang.org/grpc@v1.32.0/server.go:859 +0xf4
created by google.golang.org/grpc.(*Server).serveStreams.func1
	/home/akachurin/go/pkg/mod/google.golang.org/grpc@v1.32.0/server.go:857 +0x2c5

Debugger finished with the exit code 0
```

--- a/opentelekomcloud/acceptance/evs/resource_opentelekomcloud_blockstorage_volume_v2_test.go
+++ b/opentelekomcloud/acceptance/evs/resource_opentelekomcloud_blockstorage_volume_v2_test.go
@@ -112,7 +112,7 @@ func TestAccBlockStorageV2Volume_policy(t *testing.T) {
 
 func testPolicyPreCheck(t *testing.T) {
 	if os.Getenv("OS_KMS_KEY") == "" {
-		t.Errorf("OS_KMS_KEY should be set for this test to existing KMS key alias")
+		t.Skipf("OS_KMS_KEY should be set for this test to existing KMS key alias")
 	}
 }
 

--- a/opentelekomcloud/services/evs/resource_opentelekomcloud_blockstorage_volume_v2.go
+++ b/opentelekomcloud/services/evs/resource_opentelekomcloud_blockstorage_volume_v2.go
@@ -291,7 +291,7 @@ OUTER:
 	if err != nil {
 		return fmterr.Errorf("error fetching tags for volume (%s): %s", v.ID, err)
 	}
-	d.Set("tags", taglist)
+	d.Set("tags", taglist.Tags)
 
 	// This is useful for import
 	if d.Get("device_type").(string) == "" {

--- a/releasenotes/notes/blockstorage-tags-8ae3c5562d66e2c5.yaml
+++ b/releasenotes/notes/blockstorage-tags-8ae3c5562d66e2c5.yaml
@@ -1,0 +1,4 @@
+---
+fixes:
+  - |
+    **[EVS]** Fix panic when using ``resource/opentelekomcloud_blockstorage_volume_v2`` (`#1249 <https://github.com/opentelekomcloud/terraform-provider-opentelekomcloud/pull/1249>`_)

